contracts/truefi2/TrueLender2.sol

```diff
@@ -399,6 +399,16 @@ contract TrueLender2 is ITrueLender2, UpgradeableClaimable {
         _distribute(recipient, numerator, denominator, msg.sender);
     }
 
+    /**
+     * @dev Allow pool to transfer all LoanTokens to the SAFU in case of liquidation
+     * @param loan LoanToken address
+     * @param recipient expected to be SAFU address
+     */
+    function transferAllLoanTokens(ILoanToken2 loan, address recipient) public override {
```

Comment:
Should this be public? It seems to me, that it could be called at any time, by anyone

Reply:
there's a require below

Reply:
should be changed to external btw

Reply:
Yes there is, but require only checks if pool is correct, does not check if the loan has defaulted